### ISSUE TITLE
The directory /usr/local/bin is not writable. 

### ISSUE THREAD

**Issue by mromero1999** [yihui/tinytex#351]
I have tried using the compute's terminal to fix the problem but it asks for a password

**Comment by yihui** [comment]
Yes, it has to ask for password since it needs root privilege to make the change to this directory.

**Comment by mromero1999** [comment]
Is the password the same one as my computer? when I try to type it, it does not allow me to.

**Comment by yihui** [comment]
Yes, it is. If the password you typed does not allow you to make the change, it is likely to be an incorrect password. Unfortunately I can't help you with that.

This issue can be safely ignored if you are using **tinytex** through R.

**Comment by mromero1999** [comment]
I am using R and it says its not writable. l tried the link solution in my terminal (not R)

**Comment by yihui** [comment]
As I said, you can safely ignore this issue if you are using R. I can't help you type the correct password.

**Comment by Kiran-Raj-Dev** [comment]
I am not using R, umm what do I do?

**Comment by yihui** [comment]
Duplicate of #24
